hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenAccountWipeHandler.java

```diff
@@ -194,9 +194,15 @@ public void handle(@NonNull final HandleContext context) throws HandleException
     @Override
     public Fees calculateFees(@NonNull final FeeContext feeContext) {
         final var op = feeContext.body();
+        final var readableTokenStore = feeContext.readableStore(ReadableTokenStore.class);
+        final var tokenType =
+                readableTokenStore.get(op.tokenWipeOrThrow().tokenOrThrow()).tokenType();
         final var meta = TOKEN_OPS_USAGE_UTILS.tokenWipeUsageFrom(fromPbj(op));
         return feeContext
-                .feeCalculator(SubType.DEFAULT)
+                .feeCalculator(
```

Comment:
Do we have to take into account if the token has custom fees and eventually use `TOKEN_FUNGIBLE_COMMON_WITH_CUSTOM_FEES` and `TOKEN_NON_FUNGIBLE_UNIQUE_WITH_CUSTOM_FEES`?

Reply:
In the feeDataMap for both `TOKEN_BURN` and `TOKEN_ACCOUNT_WIPE` only subTypes available are `TOKEN_FUNGIBLE_COMMON` and `TOKEN_NON_FUNGIBLE_UNIQUE`. So judging by this I'd say we don't need to worry about custom fees. But won't hurt for someone else to confirm, just to be sure.